docs/pages/client/map.mdx

```diff
@@ -41,12 +41,20 @@ import { Callout } from 'nextra-theme-docs'
 
 ### Clustering Options
 
+Most these options are up for interpretation on which results are best for your area. Some areas and MITM clients might prefer one result over another so it is recommended you do some testing. If you want a good starting point try **Brute Force**.
+
 - Min Points
   - Minimum number of points that a cluster should have
-- Fast
-  - Uses an exponentially faster algorithm but returns worse results
-- 🚧 Only unique 🚧
-  - Counts min points by unique points instead of all points in a Cluster
+- Cluster Mode
```

Comment:
I should probably just combine cluster and route modes with a "Order By" drop down that has a "none" option... but for now, this is a good explanation.